libs/damap/src/lib/services/infoLabel.service.ts

```diff
@@ -0,0 +1,116 @@
+import { Injectable } from '@angular/core';
+import * as template from '../../assets/i18n/templates/en.json';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class InfoLabelService {
+  constructor() {}
+
+  templateData = template;
+
+  getInfo(index: number) {
+    let info;
+
+    switch (index) {
+      case 0:
+        info = {
+          gretting: this.templateData.dmp.steps.project.greeting,
+          summaryLine: this.templateData.dmp.steps.project.summaryLine,
+          instructions: this.templateData.dmp.steps.project.instructions,
+          stepNumber: 1,
+        };
```

Comment:
Probably makes sense to define an interface for the info. Allows for type hints and easier to detect changes.